.github/workflows/test.yml

```diff
@@ -1,7 +1,9 @@
-name: Tester
+name: Test
 
 on:
   push:
+    branches:
+      - main
```

Comment:
Don't we want to run the test on the commits that we push to our own branch?

Reply:
This runs on _pushes_ to `main`, but will still run on all pull requests. So as long as you open a PR (either a full PR or a draft), it will work like it used to. For instance, see this PR; there are multiple check marks throughout the PR, for each separate new commit I pushed to this PR.

This has two benefits: (1) tests won't be run on random orphan branches we create that we never start PRs from; (2) it won't run the same CI action twice when we open a PR, as it currently does (according to the configuration on `main`).